README.md

```diff
@@ -69,6 +69,16 @@ brew tap anchore/syft
 brew install syft
 ```
 
+**Container Image**
+
+Example of how to use Syft scratch image:
```

Comment:
```suggestion
Example of how to use Syft with docker:
```

Reply:
(in this way we just use the bolded title/section to hint at the installation mode)